easybuild/easyconfigs/e/Extrae/Extrae-3.8.0-foss-2020b.eb

```diff
@@ -0,0 +1,33 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+name = 'Extrae'
+version = '3.8.0'
+
+homepage = 'https://www.bsc.es/computer-sciences/performance-tools'
+description = """Extrae is the core instrumentation package developed by
+the Performance Tools group at BSC. Extrae is capable of instrumenting
+applications based on MPI, OpenMP, pthreads, CUDA1, OpenCL1, and StarSs1
+using different instrumentation approaches. The information gathered by
+Extrae typically includes timestamped events of runtime calls,
+performance counters and source code references. Besides, Extrae
+provides its own API to allow the user to manually instrument his or her
+application."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
```

Comment:
@robert-mijakovic can this be moved down to `gompi`?

Reply:
@migueldiascosta Moved! Thanks for suggesting it. I just based it on the previously existing easyconfig and didn't think about it.